oct/cli/config/show.py

```diff
@@ -72,4 +72,4 @@ def print_options(options):
     """
     max_length = max([len(repr(key)) for key in options])
```

Comment:
Alright.

BTW, there's no need to create a temporary list with the list comprehension, a generator expression will do (without the intermediate list):

```python
max_length = max(len(repr(key)) for key in options)
```